test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -259,9 +265,24 @@ synchronized void execute(String... args) {
             + String.join(" ", args)
             + " --config-dir="
             + configDirectory.toFile().getAbsolutePath();
-    logger.debug("Executing: " + command);
+
+    execute(CommandLine.parse(command));
+  }
+
+  synchronized void executeUnsanitized(String... args) {
+    String command = "ccm ";
 
     CommandLine cli = CommandLine.parse(command);
+    for (String arg : args) {
+      cli.addArgument(arg, false);
+    }
+    cli.addArgument("--config-dir=" + configDirectory.toFile().getAbsolutePath());
+
+    execute(cli);
+  }
+
+  private void execute(CommandLine cli) {
+    logger.debug("Executing: " + cli);
```

Comment:
The log can no longer be copy-pasted to the command line. If that's a problem, we can revisit later.